demo/midgard-sdk/src/utils/index.ts

```diff
@@ -3,3 +3,4 @@ export * from "./ledger-state.js";
 export * from "./linked-list.js";
 export * from "./mpt.js";
 export * from "./state-queue.js";
+export * from "./deposit.js";
```

Comment:
Same as earlier comment:
```suggestion
export * from "./user-events/deposit/index.js";
```